updatecli/weekly.d/ratings.yaml

```
@@ -0,0 +1,54 @@
+---
+title: Bump Rating Docker image version
```

```suggestion
title: Bump `jenkinsciinfra/rating` Docker image version
```
(Could/should? be in another PR)